packages/utils/node-resolver-core/test/resolver.js

```diff
@@ -297,8 +297,8 @@ describe('resolver', function () {
             aboveFilePath: path.join(rootDir, 'foo.js'),
           },
           {
-            fileName: 'package.json',
-            aboveFilePath: require.resolve('browserify-zlib'),
+            fileName: 'node_modules/browserify-zlib',
```

Comment:
`invalidateOnFileCreate` can be used to watch for folder creations (because `node_modules/browserify-zlib` would never be a regular file in practice)?